skrub/_repr.py

```diff
@@ -101,3 +101,9 @@ def doc_link_url_param_generator(estimator):
         "estimator_module": estimator_module,
         "estimator_name": estimator_name,
     }
+
+
+class _SkrubHTMLDocumentationLinkMixin(_HTMLDocumentationLinkMixin):
```

Comment:
even when the scikit-learn fix is released, we will most likely want this mixin to avoid repeating these 3 attributes everywhere. so in the end all we will have to remove is the `HTMLDocumentationLinkMixin` which is confined to this module.

so after all I am +1 for this PR